easybuild/easyconfigs/b/bat/bat-0.3.3-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Bundle'
+
+name = 'bat'
+version = '0.3.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/bat'
+description = """The BAT Python package supports the processing and analysis of Bro data with Pandas,
+scikit-learn, and Spark"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
```

Comment:
please move down, right above `exts_list` (and drop the comment, it's clear without it imho)